P5/Source/Specs/att.datcat.xml

```diff
@@ -7,51 +7,221 @@ $Date$
+      of speech</q> is abbreviated as <q>POS</q>, and the value, <q>common noun</q> is symbolized by <q>NN</q>. The
+      entire name-value pair is encoded by means of the element <gi>f</gi>. In TEI XML, that element
+      acts as the container, labeled with the <att>name</att> attribute. Its contents may be complex
+      or simple. In the case at hand, the content is the symbol <q>NN</q>.</p>
+    <p>The <att>datcat</att> attribute relates the feature <hi rend="italic">name</hi> (i.e., the key) to the data
+      category <q>part of speech</q>, while the attribute <att>valueDatcat</att> relates the feature <hi
+        rend="italic">value</hi> to the data category <soCalled>common noun</soCalled>. Both these data categories
+      should be defined in an external and preferably open reference taxonomy or ontology.</p>
+    <egXML xmlns="http://www.tei-c.org/ns/Examples">
+      <fs>
+        <f name="POS" datcat="http://hdl.handle.net/11459/CCR_C-396_5a972b93-2294-ab5c-a541-7c344c5f26c3">
+          <symbol valueDatcat="http://hdl.handle.net/11459/CCR_C-1256_7ec6083c-23d4-224d-6f94-eecbe6861545" value="NN"/>
+        </f>
+        <!-- ... -->
+      </fs>
+    </egXML>
+    <p><q>NN</q> is the symbol for common noun used e.g. in the <ref
+        target="http://ucrel.lancaster.ac.uk/claws7tags.html">CLAWS-7 tagset</ref> defined by the
+      University Centre for Computer Corpus Research on Language at the University of Lancaster. The
+      very same data category used for tagging an early version of the British National Corpus, and
+      coming from the so-called <ref target="http://www.natcorp.ox.ac.uk/docs/c5spec.html">BNC Basic
+        (C5) tagset</ref>, uses the symbol <q>NN0</q> (rather than <q>NN</q>). Making these values semantically
+      interoperable / interchangeable would be extremely difficult without a human expert if they
+      were not anchored in a single point of an established reference taxonomy of morphosyntactic
+      data categories. In the case at hand, the string <q><ref
+        target="http://hdl.handle.net/11459/CCR_C-1256_7ec6083c-23d4-224d-6f94-eecbe6861545"
+        >http://hdl.handle.net/11459/CCR_C-1256_7ec6083c-23d4-224d-6f94-eecbe6861545</ref></q> is both
+      a persistent identifier of the data category in question, as well as the address of the place where a shared definition
+      of <q>common noun</q> is provided.</p>
+    <p>While the symbols <q>NN</q>, <q>NN0</q>, and many others (often coming from languages other than
+      English) are implicitly members of the container category <q>part of speech</q>, it is sometimes
+      useful not to rely on such an implicit relationship but rather use an explicit identifier
+      for that data category, to distinguish it from other morphosyntactic data categories, such
+      as <soCalled>gender</soCalled>, <soCalled>tense</soCalled>, etc. For that purpose, the above example uses the <att>datcat</att>
```

Comment:
One tradition has been to capitalize them. I am not sure it would be accepted, so I looked for something that highlights them without scare quotes. I'll try them without any markup, then.